### PR TITLE
Add auto completion for mdi icons to code editor

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by piitaya — DISMISSED** [review]
Cool 🙂 
Ahah @bramkragten, you are too fast for me.
I was doing the same 😅 Do you think it can be good to add custom icon support?

**Comment by bramkragten** [comment]
Yeah, wanted to add `mdi` first, custom support could be added after :-)

**Comment by piitaya** [comment]
I didn't think it was even possible to add the preview of the icon. 🤩

**Review by zsarnett — DISMISSED** [review]

File: build-scripts/bundle.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ module.exports.emptyPackages = ({ latestBuild, isHassioBuild }) =>
       require.resolve(
         path.resolve(paths.polymer_dir, "src/components/ha-icon-picker.ts")
       ),
+    isHassioBuild && path.resolve(paths.polymer_dir, "build/mdi/iconList.json"),
```
Comment (zsarnett) [inline build-scripts/bundle.js:40]:
```suggestion
    isHassioBuild && path.resolve(paths.polymer_dir, "../../build/mdi/iconList.json"),
```

**Review by zsarnett — APPROVED** [review]

**Pull request merged**